samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/BasemapsLayerLists.razor

```diff
@@ -0,0 +1,73 @@
+﻿@page "/basemap-layer-lists"
+@using dymaptic.GeoBlazor.Core.Components.Widgets.LayerList
+
+
+@inject IJSRuntime JsRuntime
+
+<PageTitle>BasemapLayerLists</PageTitle>
+<h1>>Basemap and Reference layerlists</h1>
+<div class="links-div">
+    <a class="btn btn-secondary" target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-LayerList.html">ArcGIS API for JavaScript Reference</a>
```

Comment:
This link is still to the `LayerList` javascript page.

Reply:
corrected